lobby/src/test/java/org/triplea/lobby/server/db/EmailLimitIntegrationTest.java

```diff
@@ -21,7 +20,7 @@
  */
 @Integration
 public class EmailLimitIntegrationTest {
-  private final Database database = new Database(TestLobbyConfigurations.INTEGRATION_TEST);
+  // private final Database database = new Database(TestLobbyConfigurations.INTEGRATION_TEST);
```

Comment:
One thing i was wondering, is this test really just verifying that we have a column length put in place? I've been wondering what exactly this test is testing

Reply:
@RoiEXLab I'm deleting this test. The note at the top is very useful to know the max length of email column is 254. When setting the length of the column, we'll have verified that. Otherwise this test is just testing SQL, and is adding a dependency on the DB schema.

Reply:
1122b00af